message/html/compose.js

```diff
@@ -77,9 +83,10 @@ exports.create = function (api) {
       textArea.value = textArea.value.slice(0, pos) + insertLink + textArea.value.slice(pos)
 
       console.log('added:', file)
-    })
+    }, opts)
```

Comment:
So is the proposal that this opts passed down the info that this is private and then the input template in core decides how to deal with that?